app/models/authentication.rb

```diff
@@ -99,4 +110,18 @@ def event_prefix
     when "ExtManagementSystem" then "ems"
     end
   end
+
+  # will be removed once moving to yaml format, no need to review method
+  def ansible_format(options = {})
+    options.merge!('name' => "example_name", 'login' => "true", 'challenge' => "true", 'kind' => ContainerDeployment::AUTHENTICATIONS_NAMES.key(authtype))
+    res = "openshift_master_identity_providers=[" + options.to_json + "]"
+    if type.instance_of?(AuthenticationHtpasswd) && !htpassd_users.empty?
+      res += "\nopenshift_master_htpasswd_users=#{htpassd_users.first.to_json}"
+    end
+    res
+  end
+
+  def ansible_config(options = {})
+    {'name' => "example_name", 'login' => "true", 'challenge' => "true", 'kind' => authtype}.merge! options
```

Comment:
on line 116 we do:
`'kind' => ContainerDeployment::AUTHENTICATIONS_NAMES.key(authtype)`
and in 125:
`'kind' => authtype`
why the two ways ?